src/org/labkey/targetedms/model/QCPlotFragment.java

```diff
@@ -138,6 +138,19 @@ public JSONObject toJSON(boolean includeLJ, boolean includeMR, boolean includeMe
                     dataJsonObject.put("CUSUMvP", plotData.getCUSUMvP());
                     dataJsonObject.put("CUSUMvN", plotData.getCUSUMvN());
                 }
+                if (includeTrailingMean)
+                {
+                    dataJsonObject.put("TrailingMean", plotData.getTrailingMean());
+                }
+                if (includeTrailingCV)
+                {
+                    dataJsonObject.put("TrailingCV", plotData.getTrailingCV());
+                }
+                if (includeTrailingMean || includeTrailingCV)
+                {
+                    dataJsonObject.put("TrailingStartDate", plotData.getTrailingStart());
+                    dataJsonObject.put("TrailingEndDate", plotData.getTrailingEnd());
```

Comment:
Do we need the end date? Won't that always be the same as the date of the last data point?

Reply:
no, we don't. removed